packages/aws-cdk/generate.sh

```diff
@@ -13,4 +13,34 @@ cat > build-info.json <<HERE
   "comment": "Generated at $(date -u +"%Y-%m-%dT%H:%M:%SZ") by generate.sh",
   "commit": "${commit:0:7}"
 }
-HERE
\ No newline at end of file
+HERE
+
+# Build noctilucent package
+(
+  # Check out the submodule if it's not there already
+  if [ ! -f "vendor/noctilucent/Cargo.toml" ]; then
+    git -C ./vendor clone https://github.com/iph/noctilucent.git
+    git -C ./vendor/noctilucent reset --hard HEAD
+    git -C ./vendor/noctilucent git fetch && git checkout 6da7c9fade55f8443bba7b8fdfcd4ebfe5208fb1
+  fi
+
+  # update the package to the pinned commit hash
+  git -C vendor/noctilucent reset --hard HEAD
+  git -C vendor/noctilucent fetch && git -C vendor/noctilucent checkout 6da7c9fade55f8443bba7b8fdfcd4ebfe5208fb1
```

Comment:
You only need one line in the if, the other two get executed no matter what, right?

Reply:
derp ill fix this